jest.config.js

```diff
@@ -10,8 +10,10 @@ module.exports = {
     "**/?(*.)+(spec|test|unit.test).+(ts|tsx|js)"
   ],
   "transform": {
+    "^.+\\.vue$": "vue-jest",
     "^.+\\.tsx?$": "ts-jest"
```

Comment:
We are no longer using `tsx` at all. Cause there's no JSX support at all.

Reply:
I tried removing it but trying to run the test without it gives: 
![image](https://user-images.githubusercontent.com/31267481/94436644-08fc8100-01e0-11eb-8471-89877e8f03df.png)